docs/UserGuide.adoc

```diff
@@ -76,19 +76,21 @@ Examples:
 === Adding a relationship between two persons: `addRelationship`
 
 Adds a relationship between two persons in the address book +
-Format: `addRelationship FROM_INDEX TO_INDEX DIRECTION`
+Format: `addRelationship FROM_INDEX TO_INDEX DIRECTION [n/NAME] [ce/CONFIDENCE_ESTIMATE]`
 
 ****
 * The indexes of the persons are based on the most recent listing shown
 * The direction of the relationship can only be `directed` or `undirected`. The direction is case-insensitive
 * The order of the indexes matters only when the direction is `directed`, as the relationship points from the person with FROM_INDEX to the person with TO_INDEX
 * At any point of time there will be at most 1 relationship between any two persons. If adding a different relationship from the existing one between two persons is attempted, upon the addition the previous relationship between these two persons will be removed.
+* `NAME` referring to the name of the relationship *can only be alphanumeric*.
+* `CONFIDENCE_ESTIMATE` refers to the confidence estimate the user gives to the relationship. `CONFIDENCE_ESTIMATE` *can only be numeric*.
 ****
 
 Examples:
 
-* `addRelationship 1 3 directed`
-* `addRelationship 2 3 undirected`
+* `addRelationship 1 3 directed ce/12.32131`
```

Comment:
This needs to be updated to reflect the new range for confidence estimates as per #125.